python/lsst/ts/cbp/component.py

```diff
@@ -99,26 +99,28 @@ def __init__(self):
         self.mask = None
         self.mask_rotation = None
         self.masks = types.SimpleNamespace(
-            mask1=types.SimpleNamespace(name="Not a mask 1", rotation=0, id=1.),
-            mask2=types.SimpleNamespace(name="Not a mask 2", rotation=0, id=2.),
-            mask3=types.SimpleNamespace(name="Not a mask 3", rotation=0, id=3.),
-            mask4=types.SimpleNamespace(name="Not a mask 4", rotation=0, id=4.),
-            mask5=types.SimpleNamespace(name="Not a mask 5", rotation=0, id=5.),
-            mask9=types.SimpleNamespace(name="Unknown Mask", rotation=0, id=9.))
+            mask1=types.SimpleNamespace(name="Not a mask 1", rotation=0, id=1.0),
+            mask2=types.SimpleNamespace(name="Not a mask 2", rotation=0, id=2.0),
+            mask3=types.SimpleNamespace(name="Not a mask 3", rotation=0, id=3.0),
+            mask4=types.SimpleNamespace(name="Not a mask 4", rotation=0, id=4.0),
+            mask5=types.SimpleNamespace(name="Not a mask 5", rotation=0, id=5.0),
+            mask9=types.SimpleNamespace(name="Unknown Mask", rotation=0, id=9.0),
```

Comment:
Please consider using integers, not floats, for IDs. It's safer -- no danger of roundoff error -- and more logical.